dynamicio/mixins.py

```diff
@@ -52,6 +53,8 @@
     "bigint": BigInteger,
 }
 
+hdf_lock = Lock()
```

Comment:
Needed as h5 files and the default readers proly use ultithreading somehow, leading to race conditions that occasionaly break the I/O process.

Reply:
Very interesting. If you have at hand, could you please paste a relevant error from a local or Airflow run? Personally, I find it handy, because I can refer to past PRs when I see similar errors.